### PR TITLE
4.0.0

### LINKED ISSUE #151: Any plans for a new release soon?
Are there any plans for new release soon, be it major, minor, or even a patch version?

### PR DESCRIPTION
**Pull request by keithamus**
This is the chore commit to release 4.0.0

Fixes #151 #192

### CODE REVIEW

**Comment by meeber** [comment]
This seems to undo a lot of recent changes to README.md

**Comment by keithamus** [comment]
Yes I'll fold them back in. Thanks

**Comment by richardpringle** [comment]
😃

File: README.md [hunk 1]
```diff
@@ -106,11 +88,9 @@ chai.request(app)
...
   .post('/user/me')
   .type('form')
-  .send({
-    '_method': 'put',
-    'password': '123',
-    'confirmPassword': '123'
-  })
+  .send({'_method': 'put'})
+  .send({'password': '123'})
+  .send({'confirmPassword', '123'})
```
Comment (barraponto) [inline README.md]: Why / when did this change?

**Comment by keithamus** [comment]
Sorry this took so long. @meeber care for another review?

**Comment by meeber** [comment]
What's the 'is-ip' dependency version change for? Assuming that's intended,
then LGTM; merge away!

**Comment by janis91** [comment]
@keithamus Will you push this release to npm, too?

**Comment by keithamus** [comment]
Yup! Just need someone to approve this. /cc @meeber @lucasfcosta

**Comment by meeber** [comment]
@keithamus What's the story behind the `is-ip` version downgrade?

**Comment by keithamus** [comment]
@meeber gimme 5 mins and I'll fix it. Getting build errors.

**Comment by keithamus** [comment]
@meeber done I think. Needed to upgrade `simplifyify` to get es6 minification.

**Comment by janis91** [comment]
@keithamus @meeber Really straight forward 👍 nice. I appreciate it, that the new version is pushed soon, because installing the package from github has a massive performance disadvantage in our CI build.

**Comment by keithamus** [comment]
Agreed @janis91! The maintainer group has been really busy with external things - e.g. moving country, starting new jobs, and so the amount we've been able to contribute has diminished a lot. We never wanted to get into this position, but it happens. We'll be better with this stuff in the future I'm sure.

**Comment by janis91** [comment]
@keithamus That's life. ;-) Don't worry.

**Comment by richardpringle** [comment]
:tada:

**Pull request merged**